Assets/MixedRealityToolkit.Extensions/Providers/Experimental/UnityAR/UnityARCameraSettings.cs

```diff
@@ -79,9 +81,10 @@ public UnityARCameraSettingsProfile SettingsProfile
             }
         }
 
-        bool isInitialized = false;
-
 #if !(WINDOWS_UWP && !ENABLE_IL2CPP)
+        private bool isSupportedConfiguration = true;
```

Comment:
If this class is both an AR foundation and non-AR foundation handling thing, and this supported configuration is about ARFoundation only, it would be worth naming it as such (Or having a comment that this is about AR foundation only)

Reply:
Otherwise it feels like this should gate the functionality of the entire class.

Reply:
i can rename the variable -> isSupportedArConfiguration